Services.Test/TestConstants.Properties.dbl

```diff
@@ -32,7 +32,17 @@ namespace Services.Test
                     Monitor.Enter(lockObject)
                     begin
                         if (instance == ^null)
-                            instance = JsonSerializer.Deserialize<TestConstants>(File.ReadAllText(Path.Combine(Environment.GetEnvironmentVariable("SOLUTIONDIR"), "Services.Test", "TestConstants.Values.json")))
+                        begin
```

Comment:
I don't love throwing exceptions when we should be checking File.Exists first. If file exists and deserialize fails i want that to bubble up to the user not silently fail.

Reply:
Fixed. See 6bf3b98fe5fcf26526643e0b1338d8a169e07e19